### ISSUE TITLE
Set alpha on the background

### ISSUE THREAD

**Issue by mbaldinazzo** [yavski/fab-speed-dial#13]
I'd like to set the alpha when menu is open like [this](http://www.androidpolice.com/wp-content/uploads/2015/03/img_550f41a7c965a-329x585.png)
**Edit**
The container of the Fab is a CoordinatorLayout, not a FrameLayout, how can I do? Thanks.

**Comment by yavski** [comment]
@mbaldinazzo this is now possible via the xml attribute _touchGuardDrawable_. Give it a drawable of your choice and that should work. Have a look at the Style sample.

**Comment by yavski** [comment]
Available in #1.0.4.

**Comment by SvyatoslavLynda** [comment]
Hello, How to make touchGuardDrawable over AppBarLayout?

```
<android.support.design.widget.CoordinatorLayout>
<android.support.design.widget.AppBarLayout
        android:layout_width="match_parent"
        android:layout_height="wrap_content"
        android:fitsSystemWindows="false"
        android:theme="@style/AppTheme.AppBarOverlay">

        <android.support.v7.widget.Toolbar
            android:id="@+id/toolbar"
            android:layout_width="match_parent"
            android:layout_height="?attr/actionBarSize"
            android:background="?attr/colorPrimary"
            android:theme="@style/AppTheme.AppBarOverlay"
            app:popupTheme="@style/AppTheme.PopupOverlay" />
    </android.support.design.widget.AppBarLayout>
<android.support.v7.widget.RecyclerView
        android:id="@+id/list"
        android:layout_width="match_parent"
        android:layout_height="match_parent"
        app:layout_behavior="@string/appbar_scrolling_view_behavior" />
<io.github.yavski.fabspeeddial.FabSpeedDial
        android:id="@+id/fabActions"
        android:layout_width="wrap_content"
        android:layout_height="wrap_content"
        android:layout_gravity="bottom|end"
        app:fabBackgroundTint="@color/primary_purple"
        app:fabGravity="bottom_end"
        app:fabMenu="@menu/menu_speed_dial"
        app:layout_behavior="myPakage.FABScrollBehavior"
        app:miniFabDrawableTint="@android:color/white"
        app:miniFabTitleTextColor="@color/primary_grey"
        app:touchGuard="true"
        app:touchGuardDrawable="@color/touchGuardDrawable" />
<android.support.design.widget.CoordinatorLayout/>
```
